Leaflet.CheapLayerAt.js

```diff
@@ -22,6 +26,42 @@ L.Map.include({
 		return this._getLayerFromDOMElement(el);
 	},
 
+	getLayersAt: function(point, y) {
+		point = L.point(point, y);
+
+		// Ignore points outside the map
+		if (!this.getSize().contains(point)) { return; }
+
+		var mapPos = this._container.getBoundingClientRect();
```

Comment:
This bit could be refactored into something common to `getLayersAt` and `getLayerAt`, e.g. `_mapPointToDocumentPoint()`